autowarefoundation/autoware.universe
# fix: fix internal door interface qos

## Description

Fix QoS of vehicle doors interface.

## Related links

**Parent Issue:**

- [TIER IV internal link](https://star4.slack.com/archives/C07DMS06H6U/p1729655528407399?thread_ts=1729644240.186299&cid=C07DMS06H6U)

## How was this PR tested?

1. Launch `ros2 launch vehicle_door_simulator vehicle_door_simulator.launch.xml`
2. Launch planning simulator
3. Check `ros2 topic echo /api/vehicle/doors/status` is published 

## Notes for reviewers

None.

## Interface changes

None.



## Effects on system behavior

None.
